wgsl/index.bs

```diff
@@ -760,7 +760,9 @@ An attribute [=shader-creation error|must not=] be specified more than once per
   </thead>
 
   <tr><td><dfn noexport dfn-for="attribute">`align`</dfn>
-    <td>[=integer literal=], yielding a positive i32
+    <td>
+        * [=AbstractInt=] or [=i32=]
+        * Must be a positive [=i32=]
```

Comment:
Only one of these bullets describes a value. The second bullet seems to completely override the first bullet.

Separately, this seems to imply that it is not restricted to a literal. We explicitly decided to push allowing creation-time expressions in attributes to post-V1 (see #2790).

Reply:
Would also suggest that if "must" is used in the text that it be used as `[=shader-creation error|Must=]`.

Reply:
Do you mean that the `i32` in the second point is overriding the `AbstractInt` in the first point?

Would `Must evaluate to a positive [=i32=]` be better?

Reply:
>Do you mean that the i32 in the second point is overriding the AbstractInt in the first point?
Right, mentioning the same type twice seems to restrict the previous restriction.

Maybe these would be better to say:
>Must be a positive [=i32=] literal

Then have a note elsewhere in the section that those literals can be abstract integers through standard conversions and link to the promotion rules.

Reply:
>Do you mean that the i32 in the second point is overriding the AbstractInt in the first point?
Right, mentioning the same type twice seems to restrict the previous restriction.

Maybe these would be better to say:
>Must be a positive [=i32=] literal

Then have a note elsewhere in the section that those literals can be abstract integers through standard conversions and link to the promotion rules.

Reply:
I thought about that, but u32 is convertible to i32. Maybe mentioning automatic conversion, but I'm not sure.

Reply:
`Must be a positive [=i32=] or [=AbstractInt=] literal` ?

Reply:
That works.
```suggestion
[=shader-creation error|Must=] be a positive [=i32=] or [=AbstractInt=] [=integer literal|literal=]
```

This applies similarly to most of the other changes too.

Reply:
done